### ISSUE TITLE
[lager] Array's dead

### ISSUE THREAD

**Issue by inducer** [illinois-scicomp/machine-shop-maintenance#53]
This means: two more disks die, and the array is done. I am running backups every for hours... so maybe we can wait for this to happen. Don't know.

cc @lukeolson

**Comment by inducer** [comment]
No redundancy remaining now. When the next disk goes, the array is done. (There's an ongoing backup to bock.)

**Comment by inducer** [comment]
Still no redundancy remaining. *plays Jaws theme* :shark:

**Comment by inducer** [comment]
Looks like it died just now.

cc @lukeolson

**Comment by inducer** [comment]
I've disconnected the array and made the backup on bock the NFS server for now. The final health state of the disks is as follows:

![disk-state-2022-01-22](https://user-images.githubusercontent.com/352067/150662631-25a290fd-1ef6-46a6-b9b1-93ca4573980b.png)

Disk 11 in enclosure 1 (off this screen) is also OK.

Sorry about the German. "Steckplatznummer" = Slot, "Fehlerhaft" = Failed, "Bereit" = Ready.

According to #51, the following disks are to be avoided because of previous issues:

- Disk 7 in Enclosure 1
- Disk 1 in Enclosure 1
- Disk 5 in Enclosure 1

I've applied color-coded stickers to the disks, but those will probably fall off with time:

![IMG_20220122_184712](https://user-images.githubusercontent.com/352067/150662697-eb1464b9-73da-494d-8a36-251e735d4404.jpg)

Interpret the color code as follows:

- pink = good drive
- orange = past zombie drive, steer clear
- yellow = failure expected
